model/link.go

```diff
@@ -19,10 +19,18 @@ import (
 )
 
 type Link struct {
-	Uri    string            `json:"uri"`
-	Expire time.Time         `json:"expire,omitempty"`
-	Method string            `json:"method,omitempty"`
-	Header map[string]string `json:"header,omitempty"`
+	Uri      string            `json:"uri" bson:"-"`
+	Expire   time.Time         `json:"expire,omitempty" bson:"expire"`
+	Method   string            `json:"method,omitempty" bson:"-"`
```

Comment:
where do we use Method and Header?

Reply:
It was added as extra metadata to the `storage` client.